src/applications/disability-benefits/2346/reducers/form2346Reducer.js

```diff
@@ -0,0 +1,41 @@
+import {
+  FETCH_DATA_SUCCESS,
+  FETCH_DATA_FAILURE,
+  UPDATE_DATA_FAILURE,
+  UPDATE_DATA_SUCCESS,
+} from '../constants';
+
+const initialState = {
+  data: [],
```

Comment:
If `data` is an array, could we update the name of this to be more specific? What is it a list of?

Reply:
@kelsonic It's the mock JSON data we're getting from vets api. Eventually it will hold the real data from the DLC.

Reply:
Interesting, so is it then an object instead of an array (e.g. a list of something)?